packages/client/src/theme.jsx

```diff
@@ -0,0 +1,61 @@
+import { createTheme } from '@mui/material/styles';
+
+// Compose theme in several steps so we can
+//   (1) use augmentColor, and
+//   (2) style components using named colors.
+export let theme = createTheme({});
+
+theme = createTheme(theme, {
```

Comment:
This could just be one:

```suggestion
export const theme = createTheme({
```

Reply:
Will fix! But I think it has to be `let` because of the theme composition thing described in the lower comment

Reply:
Actually I forgot - `theme.palette.augmentColor`  doesn't work in the latter part unless I define `theme` first, so I had to do it like this